src/content/en/2021/resource-hints.md

```diff
@@ -1,20 +1,459 @@
+
+<figure markdown>
+| host                       | dns-prefetch | preconnect | preload | Total       | 
+|----------------------------|--------------|------------|---------|-------------| 
+| adservice.google.com       | 0.25%        | 0.37%      | 42.72%  | 43.34%      | 
+| fonts.gstatic.com          | 0.82%        | 25.45%     | 0.74%   | 27.01%      | 
+| fonts.googleapis.com       | 14.09%       | 4.59%      | 3.05%   | 21.72%      | 
+| s.w.org                    | 20.00%       | 0.25%      | -       | 20.25%      | 
+| cdn.shopify.com            | -            | 1.82%      | 12.16%  | 13.98%      | 
+| www.google-analytics.com   | 1.27%        | 4.05%      | 0.17%   | 5.49%       | 
+| www.googletagmanager.com   | 1.94%        | 3.14%      | 0.25%   | 5.32%       | 
+| siteassets.parastorage.com | -            | -          | 5.24%   | 5.24%       | 
+| images.squarespace-cdn.com | -            | 4.51%      | -       | 4.51%       | 
+| ajax.googleapis.com        | 2.30%        | 1.79%      | 0.36%   | 4.45%       | 
+
+<figcaption>{{ figure_link(caption="Most popular third parties connected to using resource hints.", sheets_gid="1254656281", sql_file="preload_host_by_url.sql") }}</figcaption>
+</figure>
+
+Analysing the table above, 42.72% of all pages which include a `preload` hint are preloading resources hosted on adservice.google.com, while s.w.org (WordPress) is the most popular domain for `dns-prefetch` and `fonts.gstatic.com` for the `preconnect` directive.
+
+<p class="note">Note: Google Fonts now includes instructions to <code>preconnect</code> to both the fonts.gstatic.com origin and fonts.googleapis.com.</p>
+
+### Native Lazy Loading
+
+Native image and iframe lazy-loading have been standardized in 2019 and since then, their adoption - especially for images - has grown exponentially. Lazy-loading refers to the technique to defer downloading a resource - in this case an image or iframe - until it is needed or visible within the viewport.
+
+`loading="lazy"` on images is supported on most major browsers. On Safari, it is marked as <a hreflang="en" href="https://bugs.webkit.org/show_bug.cgi?id=200764">in progress</a> and is available behind a flag, but not yet enabled by default. Lazy-loading of iframes has similar support, although it is <a hreflang="en" href="https://bugzilla.mozilla.org/show_bug.cgi?id=1622090">not yet supported on Firefox</a>.
+
+Browsers which do not support the loading attribute, will simply ignore it - making it safe to add without unwanted side-effects. JavaScript based alternatives, such as <a hreflang="en" href="https://github.com/aFarkas/lazysizes">lazysizes</a> may still be used, however considering that full browser support is around the corner, it may not be worth adding to a project at this stage.
```

Comment:
```suggestion
Browsers which do not support the `loading` attribute, will simply ignore it - making it safe to add without unwanted side-effects. JavaScript based alternatives, such as <a hreflang="en" href="https://github.com/aFarkas/lazysizes">lazysizes</a> may still be used, however considering that full browser support is around the corner, it may not be worth adding to a project at this stage.
```